### PR TITLE
More usable interface for log plotting

### PR DESCRIPTION
**Pull request by alecokas**
The average KL per latent dimension plot can now be displayed or saved. One must access this functionality through the new ```create_plots.py``` script.

Options for PDF and PNG are also included.

### CODE REVIEW

File: create_plots.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import argparse
```
Comment (YannDubs) [inline create_plots.py]: that's starting to be a lot of files with a single function. try to put things otgether. 

For example, we already have a whole `viz` directory (which honestly we could put in utils=

why is this cli not with `run_experiments.py` ?

**Comment by YannDubs** [comment]
why is this PR not with the other one ? 
PS: it's fine don't change the PR and merge stuff, I was just woderning cause it seems to just be the continuation of the other.

Haven't tested anything but feel free to merge.

As a geenral comment plase keep the number of files at the minimum and try to keep things together. FOr example it's not cler to me why log plotter is in utils and not viz (although I actually think all of the viz could be in util). By putting all functions with similar roles together it makes it easier to follow and understand :)
ALso I don't think we need 3 different cli for one simple directory

**Comment by alecokas** [comment]
> why is this PR not with the other one ?
> PS: it's fine don't change the PR and merge stuff, I was just woderning cause it seems to just be the continuation of the other.
> 
> Haven't tested anything but feel free to merge.
> 
> As a geenral comment plase keep the number of files at the minimum and try to keep things together. FOr example it's not cler to me why log plotter is in utils and not viz (although I actually think all of the viz could be in util). By putting all functions with similar roles together it makes it easier to follow and understand :)
> ALso I don't think we need 3 different cli for one simple directory
> 
> I.e. I love modularity but having in different files makes it hard to follow

I guess I envisioned it as a separate functionality in the sense that we could do all plotting from here, so a separate PR would be required. I certainly don't feel strongly about it though

**Comment by alecokas** [comment]
Merging after making the suggested changes.
